app/src/main/java/com/rorpage/purtyweather/util/WeatherIconsUtil.kt

```diff
@@ -9,11 +9,10 @@ class WeatherIconsUtil(val context: Context) {
 
     fun getIcon(iconFileName: String): Drawable? {
         Timber.v("Icon Name $iconFileName")
-        return ContextCompat.getDrawable(context, context.resources.getIdentifier("wi_$iconFileName", "drawable", context.packageName))
+        return ContextCompat.getDrawable(context, getIconId(iconFileName))
```

Comment:
This is the only change that wasn't obviously a ktlint thing. Do you remember if it was just an IDE "quick fix" you were able to implement? It seems OK to me. Just curious. :)

Reply:
No, it wasn't a quick fix. I was looking at this file because of a Lint issue and I realized the duplication.